dashboard/config/blocks/Dancelab/Dancelab_ai.json

```diff
@@ -20,6 +20,26 @@
         "name": "VALUE",
         "customInput": "danceAi"
       }
-    ]
+    ],
+    "jsonConfig": {
```

Comment:
The Blockly JSON config. For now this is defined within the `config` property just to avoid having to change the schema of the `block` model.